mets.md

```diff
@@ -160,6 +199,83 @@ encodings of the same page.
 </mets:structMap>
 ```
 
+## OCR-D structMap
+
+A METS can contain different `structMap`. These are defined by the `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`>) described in more detail.	
```

Comment:
```suggestion
A METS may contain different `mets:structMap` entries, differentiated by their `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`). Some of these are mandatory within OCR-D.
```

Reply:
A METS-file may contain different `mets:structMap` entries, differentiated by their `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL, ...`). 

- A `mets:structMap` with `TYPE="PHYSICAL"` is mandatory.  

- The *logical* document structure detected by *library or archive* MUST be described by `TYPE="LOGICAL"`.

- The *logical* document structure detected by *OCR-D software* MUST be described by `TYPE="OCR-D-LOGICAL"`.

Reply:
```suggestion
A METS may contain different `mets:structMap` entries, differentiated by their `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL, ...`). 
* A `mets:structMap` with `TYPE="PHYSICAL"` is mandatory.
* The _logical_ document structure detected by _library or archive_ MUST be described by `TYPE="LOGICAL"`.
* The _logical_ document structure detected by _OCR-D software_ MUST be described by `TYPE="OCR-D-LOGICAL"`.
```

Reply:
Maybe you could add a hyperlink to the METS profile for logical structMap we want to use here?